code/network/stand_gui.cpp

```diff
@@ -170,16 +170,16 @@ void std_gen_set_text(char *str, int field_num)
 	}
 
 	// otherwise set the text
-	ctrl = GetDlgItem(Multi_gen_dialog, (int)MAKEINTRESOURCE(IDC_FIELD1));
+	ctrl = GetDlgItem(Multi_gen_dialog, (int)(size_t)MAKEINTRESOURCE(IDC_FIELD1));
```

Comment:
The change is needed, I needed it before when I tried to do the same. However you should use `uintptr_t` since that is an integer type that can hold a pointer. `size_t` isn't the right type for this.